### PR TITLE
Device signal service working impl

### PR DESCRIPTION
**Pull request by cometman**

### CODE REVIEW

File: NeuroIDAdvancedDevice.xcodeproj/xcuserdata/clay.xcuserdatad/xcschemes/xcschememanagement.plist [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+	<key>SchemeUserState</key>
+	<dict>
+		<key>NeuroIDAdvancedDevice.xcscheme_^#shared#^_</key>
+		<dict>
+			<key>orderHint</key>
+			<integer>3</integer>
+		</dict>
+	</dict>
+</dict>
+</plist>
```
Comment (kevinsites) [inline NeuroIDAdvancedDevice.xcodeproj/xcuserdata/clay.xcuserdatad/xcschemes/xcschememanagement.plist]: Is this file necessary? It seems to be related to your local machine `xcuserdata/clay....` ?

**Review by quacke-nid — COMMENTED** [review]
Is there a limitation in swift testing frameworks requiring that DeviceSignal be subclassed?  All things being equal, the ios sdk should test the public interface of the NeuroIDADV and not an internal implementation.

**Review by quacke-nid — DISMISSED** [review]

**Review by kevinsites — DISMISSED** [review]

**Review by kevinsites — APPROVED** [review]

**Review by priya-xavier — APPROVED** [review]
